backend/app/main.py

```diff
@@ -116,6 +116,16 @@
     tags=["extractors"],
     dependencies=[Depends(get_current_username)],
 )
+api_router.include_router(
+    extractors.router,
```

Comment:
You newly added routes are in routers/datasets.py; but this new prefix is importing routes from extractors.
So i can actually access your routes under the `/dataset/{id}/summary.jsonld` but not here

I'd recommend you separate them out in a new router file (e.g. summary.py) and include them here. `summary.router`

Reply:
that was left over, thanks, will skip the tags for now, and take your other advice to put those routes in another file